docs/platforms/apple/common/install/cocoapods.mdx

```diff
@@ -15,3 +15,5 @@ end
 ```
 
 Afterwards run `pod install`.
+
+In case you are using Xcode 14 or later, you might have to disable the "Enable User Script Sandbox" option in the target settings by setting it to `NO`. See <PlatformLink to="/troubleshooting#sandbox-error-using-cocoapods">Troubleshooting</PlatformLink> for more information.
```

Comment:
`m`: I think we should briefly mention the error here as well cause devs tend to overlook this.
```
error: Sandbox: rsync.samba(42924) 
```

Reply:
```suggestion
In case you are using Xcode 14 or later, your project might fail to build due to `error: Sandbox: rsync.samba(42924)`. To resolve this you have to disable the "Enable User Script Sandbox" option in the target settings by setting it to `NO`. See <PlatformLink to="/troubleshooting#sandbox-error-using-cocoapods">Troubleshooting</PlatformLink> for more information.
```